### PR TITLE
[IMP] web: update to fontawesome 4.7.0 to 5.10.1

### PR DESCRIPTION
**Pull request by kbapt**
[IMP] web: update to fontawesome 4.7.0 to 5.10.1

FontAwesome 5 introduced new names for some icons as described on
https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4#name-changes
    
TaskID: 2050241

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/_extensions/odoo_ext/layout.html [hunk 1]
```diff
@@ -328,9 +328,9 @@ <h1> {{ meta.get('main-title', title) }} </h1>
       <div class="container">
         <a class="small" href="http://www.odoo.com/page/website-builder">Website made with <span class="o_logo o_logo_inverse o_logo_15"></span></a>
         <div class="social-links pull-right">
-          <a href="http://www.odoo.com/web/about/facebook"><i class="fa fa-facebook"></i></a>
-          <a href="http://www.odoo.com/web/about/twitter"><i class="fa fa-twitter"></i></a>
-          <a href="http://www.odoo.com/web/about/linkedin"><i class="fa fa-linkedin"></i></a>
+          <a href="http://www.odoo.com/web/about/facebook"><i class="fab  fa-font-awesomecebook-f-f"></i></a>
```
Comment (mreficent) [inline doc/_extensions/odoo_ext/layout.html]: I think something wrong is here

**Comment by Yenthe666** [comment]
This PR will also close off https://github.com/odoo/odoo/pull/35826

**Comment by yelizariev** [comment]
Will we see it in Odoo 14?

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo r+

**Comment by tivisse** [comment]
@robodoo p=1

**Pull request closed without merging**